shlr/windbg/windbg.c

```diff
@@ -256,11 +259,15 @@ int windbg_wait_packet(WindCtx *ctx, const uint32_t type, kd_packet_t **p) {
 			R_FREE (pkt);
 		}
 		// Try to read a whole packet
-		ret = kd_read_packet (ctx->io_ptr, &pkt);
+		ret = kd_read_packet (ctx->desc, &pkt);
 		if (ret != KD_E_OK || !pkt) {
 			break;
 		}
 
+		if (pkt->type == KD_PACKET_TYPE_UNUSED) {
+			continue;
```

Comment:
This will exaust `retries` if theres many unused packets unread

Reply:
@abcSup please don't forget about this comment.

Reply:
thanks for the reminder. I have added code to increase `retries` for unused KD packet.